cachito/workers/tasks.py

```diff
@@ -15,3 +18,22 @@
 def add(x, y):
     """Add two numbers together to prove Celery works"""
     return x + y
+
+
+@app.task
+def fetch_source(url, ref):
+    """
+    Fetch the application source code that was requested and put it in long-term storage.
+
+    :param str url: the source control URL to pull the source from
+    :param str ref: the source control reference
+    """
+    log.info('Fetching the source from "%s" at reference "%s"', url, ref)
+    try:
+        # Default to Git for now
+        scm = Git.create_from_url(url, ref)
```

Comment:
`Git(url, ref)` ?

Reply:
Oops, I forgot to update this after I changed it.